CHANGELOG

```diff
@@ -1,4 +1,4 @@
-0.3.0
+0.2.1
```

Comment:
I know, but since you labeled it `enh:`, I thought you would consider this an enhancement/feature rather than a fix 🙈 Both is fine with me.

Reply:
Yes, I always use `feat: ...` for new features. I might have read that in a semantic changelog/versioning blog post, but I cannot find it right now.